### ISSUE TITLE
add when user joins a project and use it to only show notifications up to that join date

### ISSUE THREAD

**Issue by timothyclemansinsea** [sagemathinc/smc#1308]
one of two cases of #631

**Comment by williamstein** [comment]
NOTE -- the full project log might not get loaded -- only the last few thousand (?) entries.  But in case the user isn't listed, just show all notifications.

**Comment by williamstein** [comment]
Yes, this is likely a good idea.    It does seem really annoying to see tons of old notifications right when you join.   

This might not be a good idea though.  Better might just be an easy way to mark all notifications from a given project read (or not).  E.g., in zulip when you join you see that there are a billion unread things.... and maybe you want to systematically catch up on some of them.  I'm labeling this "unclear".

Also, note that we never ever get user complaints about this.
